release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3330,6 +3330,78 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.6 --pullspecs
+----
+
+[id="ocp-4-16-6-enhancements_{context}"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-16-6-enhancements-_{context}"]
+===== Ingress Controller certificate expiration dates collected
+
+* The Insights Operator now collects information about all Ingress Controller certificate expiration dates. The information is put into a JSON file in the path `aggregated/ingress_controllers_certs.json`. (link:https://issues.redhat.com/browse/OCPBUGS-37671[*OCPBUGS-37671*])
+
+[id="ocp-4-16-6-enhancements-debug-log_{context}"]
+===== Enabling debug log levels
+
+* Previously, you could not control log levels for the internal component that selects IP addresses for cluster nodes. With this release, you can now enable debug log levels so that you can either increase or decrease log levels on demand. To adjust log levels, you must create a config map manifest file with a configuration analogous to the following:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  enable-nodeip-debug: "true"
+kind: ConfigMap
+metadata:
+  name: logging
+  namespace: openshift-vsphere-infra
+# ...
+----
+(link:https://issues.redhat.com/browse/OCPBUGS-35891[*OCPBUGS-35891*])
+
+[id="ocp-4-16-6-bug-fixes_{context}"]
+==== Bug fixes
+
+*  Previously, installer-created subnets were being tagged with `kubernetes.io/cluster/<clusterID>: shared`. With this release, subnets are now tagged with `kubernetes.io/cluster/<clusterID>: owned`. (link:https://issues.redhat.com/browse/OCPBUGS-37510[*OCPBUGS-37510*])
+
+* Previously, the same node was queued multiple times in the draining controller which, caused the the same node to be drained twice. With this release, the a node will only be drained once. (link:https://issues.redhat.com/browse/OCPBUGS-37470[*OCPBUGS-37470*])
+
+* Previously, cordoned nodes in machine config pools (MCPs) with higher `maxUnavailable` than unavailable nodes might be selected as an update candidate. With this release, cordoned nodes will never be queued for an update. (link:https://issues.redhat.com/browse/OCPBUGS-37460[*OCPBUGS-37460*])
+
+* Previously, oc-mirror plugin v2, when running behind proxy with the system proxy configuration set, would attempt to recover signatures for releases without using the system proxy configuration. With this release, the system proxy configuration is taken into account during signature recovery as well and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37445[*OCPBUGS-37445*])
+
+* Previously, an alert for OVNKubernetesNorthdInactive would not fire in circumstances where it should fire. With this release, the issue is fixed so that the alert for OVNKubernetesNorthdInactive fires as expected. (link:https://issues.redhat.com/browse/OCPBUGS-37362[*OCPBUGS-37362*])
```

Comment:
Not sure but this string looks like something that would appear in a CLI output, should it be in backticks?
```suggestion
* Previously, an alert for `OVNKubernetesNorthdInactive` would not fire in circumstances where it should fire. With this release, the issue is fixed so that the alert for `OVNKubernetesNorthdInactive` fires as expected. (link:https://issues.redhat.com/browse/OCPBUGS-37362[*OCPBUGS-37362*])
```